google/cloud/spanner/row.h

```diff
@@ -0,0 +1,179 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_SPANNER_ROW_H_
+#define GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_SPANNER_ROW_H_
+
+#include "google/cloud/spanner/version.h"
+#include <cstdint>
+#include <string>
+#include <tuple>
+#include <utility>
+
+namespace google {
+namespace cloud {
+namespace spanner {
+inline namespace SPANNER_CLIENT_NS {
+/**
+ * The `Row<Ts...>` class template represents a heterogeneous set of C++ values.
+ *
+ * The values stored in a row may have any of the valid Spanner types described
+ * in value.h. Each value in a row is identified by its column index, with the
```

Comment:
nit: s/in value.h/can be stored in a `Value`/

That way you get a link to the type, and sticks if we move the type to some other header (unlikely I know).

Reply:
Cool. Didn't know that. Thanks.